### ISSUE TITLE
general readability

### ISSUE THREAD

**Issue by musikBear** [LMMS/lmms#2282]
in context of several tickets (#1839 , #1935 among others) The issue of general readability need focus.
It is a *huge* thing for all designs, and play a vital role in software usecases. 
Low readability causes headache and can lead to eye-deceases, so the subject goes under ergonomic usability of software.
Readability often clashes with design.
Coolness is not always the same as readability -Quite often the opposite..
Now, i should not gabble on about these things, some are most likely already annoyed   
In my point of view lmms could tackle readability in 2 completely different ways:
* General design
* Theming

Imo there does need to be some changes in general-design to enhance readability, but the wishes for nice design, and a fluent color-scheme, weighs a lot. Many design related tickets has stressed this over and over.
Here is a couple of imgs where lmms is in need of general design-changes
![readblocknames](https://cloud.githubusercontent.com/assets/6662066/9382481/7b593216-4741-11e5-8e09-63d21b9677e1.jpg)

![grimmetal](https://cloud.githubusercontent.com/assets/6662066/9382495/a63f65f4-4741-11e5-9fd6-53f416875786.png)

So the other option Theming!
If all the issues where lmms stands weak in readability are theme-able, then a lot is reached! I feel that the right way to go, is themes, because that will lead us around a huge debate about general design and cool look. All that (pie-fighting) is transparent, if readability is done as theming.

So in conclusion:
Add to this thread everything that you see as an issue of 
* visibility (like text obscuring or being obscured
* readability (color differences between foreground and background. Font-size/type etc

Then lets take a constructive discussion around the workload versus the benefit of getting theming possibilities, for these cases.

**Comment by tresf** [comment]
Just focus on the problem(s) and don't TL;DR this please.

We have enough "better theme" threads and they'll be open for eternity.  Let's focus on the exact problems at hand, make a checklist and we'll go from there.  People solving these problems don't need a lecture on what good UI design is.

**Comment by tresf** [comment]
Please edit your original post to be clear and concise as to the problem, and we'll categorize it appropriately.  Thanks.

**Comment by tresf** [comment]
@musikBear Please edit your original post to be clear and concise as to the problem, and we'll categorize it appropriately. Thanks.

**Comment by tresf** [comment]
![grimmetal](https://cloud.githubusercontent.com/assets/6662066/9382495/a63f65f4-4741-11e5-9fd6-53f416875786.png)
Duplicate of #2259, why post this here too?

@musikBear thanks for cleaning up the original post, but you still fail to mention a specific problem and haven't fixed the title.

The problem in your original bug report is "Piano Roll description / tooltip hard to read", however your title says "General Readiblity".

"General Readiblity" is ok too, if you take the time to document the places that are hard to read, but you need to choose one or the other.

**Comment by Umcaruje** [comment]
FYI: I turned this into a checklist, since I solved one of the issues in my recent PR.
